<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
background-color: rgb(42, 39, 41);</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Сумма налога:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Сумма депозита:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_sum_deposit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ui_sum_nalog">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_time_deposit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Изъять:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_procent">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>День:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_res">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>420</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid black;
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(253, 159, 9);
}
QPushButton:pressed {
    background-color: rgb(202, 125, 6);
    border: 2px solid black;
}
</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Срок вклада:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>440</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>10</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Список пополнений/
изъятий:</string>
   </property>
  </widget>
  <widget class="QLabel" name="err">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 35pt;
background-color: rgb(42, 39, 41);
color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>дней</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Налоговая ставка:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_nalog">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 30pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Тип выплаты</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="ui_capital_procent">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>300</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов (F1)</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ui_type">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 18pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <item>
    <property name="text">
     <string>Каждый день</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Каждый месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Каждый квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Каждые полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Каждый год</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Начисленные %:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Сумма на вкладе</string>
   </property>
  </widget>
  <widget class="QLabel" name="ui_pay_procent">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="ui_last_deposit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>к концу срока:</string>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_day_deposit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_sum_day_deposit">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>100</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Сумма:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>День:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_day_pay">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>200</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ui_sum_day_pay">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Сумма:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>260</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>у.е.</string>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 4px solid grey;
   border-radius: 0px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>капитализации:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid black;
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(95, 92, 93);
}
QPushButton:pressed {
    background-color: rgb(161, 160, 159);
    border: 2px solid black;
}
</string>
   </property>
   <property name="text">
    <string>Пополнение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_take">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>360</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
   border: 1px solid black;
   border-radius: 0px;
   font: 25pt;
   background-color: rgb(95, 92, 93);
}
QPushButton:pressed {
    background-color: rgb(161, 160, 159);
    border: 2px solid black;
}
</string>
   </property>
   <property name="text">
    <string>Изъятие</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="transactions">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">   border: 1px rgb(42, 39, 41);
   border-radius: 0px;
   font: 15pt;
   background-color: rgb(68, 65, 66);</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px rgb(42, 39, 41);
border-radius: 1px;
font: 19pt;
background-color: rgb(42, 39, 41);</string>
   </property>
   <property name="text">
    <string>Пополнить:</string>
   </property>
  </widget>
  <zorder>label_25</zorder>
  <zorder>label_8</zorder>
  <zorder>label_13</zorder>
  <zorder>label</zorder>
  <zorder>ui_sum_deposit</zorder>
  <zorder>ui_sum_nalog</zorder>
  <zorder>ui_time_deposit</zorder>
  <zorder>label_5</zorder>
  <zorder>ui_procent</zorder>
  <zorder>label_12</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_res</zorder>
  <zorder>label_2</zorder>
  <zorder>label_14</zorder>
  <zorder>label_16</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>ui_nalog</zorder>
  <zorder>label_10</zorder>
  <zorder>ui_capital_procent</zorder>
  <zorder>ui_type</zorder>
  <zorder>label_11</zorder>
  <zorder>label_17</zorder>
  <zorder>ui_pay_procent</zorder>
  <zorder>ui_last_deposit</zorder>
  <zorder>label_18</zorder>
  <zorder>line_7</zorder>
  <zorder>line_8</zorder>
  <zorder>line_9</zorder>
  <zorder>line_10</zorder>
  <zorder>label_19</zorder>
  <zorder>label_20</zorder>
  <zorder>ui_day_deposit</zorder>
  <zorder>ui_sum_day_deposit</zorder>
  <zorder>label_15</zorder>
  <zorder>label_21</zorder>
  <zorder>ui_day_pay</zorder>
  <zorder>ui_sum_day_pay</zorder>
  <zorder>label_22</zorder>
  <zorder>label_23</zorder>
  <zorder>label_24</zorder>
  <zorder>line_11</zorder>
  <zorder>pushButton_add</zorder>
  <zorder>pushButton_take</zorder>
  <zorder>err</zorder>
  <zorder>transactions</zorder>
  <zorder>label_26</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
